client/src/pages/index.js

```diff
@@ -0,0 +1,24 @@
+export { default as CarbonFootprint } from "./CarbonFootprint/CarbonFootprint.js";
+export { default as EnergyConsumption } from "./CarbonFootprint/EnergyConsumption/EnergyConsumption.js";
+
+export { default as CommutingHabits } from "./CommutingHabits/CommutingHabits.js";
+export { default as EBikeCycling } from "./CommutingHabits/EBikeCycling/EBikeCycling.js";
+export { default as PublicTransport } from "./CommutingHabits/PublicTransport/PublicTransport.js";
+export { default as RideSharing } from "./CommutingHabits/Ridesharing/Ridesharing.js";
```

Comment:
Since we are in here right now, can you change this to RideSharing and update all directories / components and file names to match that? @xinyi-dev

Reply:
sure, will work on it!